app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -371,9 +371,9 @@ sealed class HomePanelCta(
 ) : Cta, ViewCta {
 
     override fun showCta(view: View) {
-        view.ctaIcon.setImageResource(image)
-        view.ctaTitle.text = view.context.getString(title)
-        view.ctaSubtitle.text = view.context.getString(description)
+        view.ctaIcon?.setImageResource(image)
```

Comment:
Q: did you see any issue here that made you use `?`?

Reply:
It's been added as there is a new layout that uses the same `showCta` function and may not have all these. Will be reverted after the experiment.